kratos/geometries/brep_curve_on_surface.h

```diff
@@ -338,6 +338,70 @@ class BrepCurveOnSurface
             mCurveNurbsInterval.GetT0(), mCurveNurbsInterval.GetT1());
     }
 
+    ///@}
+    ///@name IsInside
+    ///@{
+
+    /// returns if rPointLocalCoordinates[0] is inside -> 1 or outside -> 0
+    int IsInsideLocalSpace(
```

Comment:
bool?

Reply:
It is actually chosen to be integer for 0 outside 1 inside and 2 on the boundary

Reply:
Furthermore, int is the same efficiency as bool.

Reply:
In either way, this is how it's defined in the base class

Reply:
2 is not specified

Reply:
added